roles/StackStorm.nginx/tasks/nginx_yum.yml

```diff
@@ -66,3 +124,31 @@
     persistent: yes
   when: ansible_facts.selinux.status == "enabled" and ansible_facts.selinux.mode == "enforcing"
   tags: nginx
+
+- name: Discover if hypervisor is EC2
+  become: yes
+  lineinfile:
+     path: /sys/hypervisor/uuid
+     line: "ec2"
+     state: present
+  check_mode: yes
+  ignore_errors: true
+  register: ec2
+  tags: nginx
+  when: (ansible_facts.os_family == 'RedHat' and ansible_facts.distribution_major_version == '8')
+
+- name: Add http and https services to firewalld
```

Comment:
I think we want to ensure firewalld is not present in ec2, right?

Reply:
In the one line install it only enabled httpd and https services if it was RH8 and there was a hypervisor/sysuuid and that did not contain ec2
I did wonder about just changing to be enable those services if firewalld was enabled, as they seemed more what was needed.